packages/wallets/src/components/Base/WalletButton/WalletButton.tsx

```diff
@@ -1,28 +1,33 @@
-import React, { ButtonHTMLAttributes, ReactElement } from 'react';
+import React, { ButtonHTMLAttributes, CSSProperties, ReactElement } from 'react';
 import classNames from 'classnames';
 import { TGenericSizes } from '../types';
 import { WalletText } from '../WalletText';
-import styles from './WalletButton.module.css';
+import './WalletButton.scss';
 
-interface WalletButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-    color?: 'black' | 'primary-light' | 'primary' | 'white';
+type TVariant = 'ghost' | 'outlined' | ('contained' & { color: WalletButtonProps['color'] });
```

Comment:
just a question, what does the ghost variant look like? Is it this button?
<img width="901" alt="Screenshot 2023-10-20 at 10 01 28 AM" src="https://github.com/binary-com/deriv-app/assets/103016120/569b9281-2979-4df8-bf22-39a064947681">

Reply:
That one is `Contained` with color of `primary-light`

Reply:
sounds like a good case for some sort of demo page with all the components and all major variants - like a design guide or something, 
as I found all the naming confusing without it being visualised somewhere,